_topics/en/fee-estimation.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Fee estimation
```

Comment:
I used "fee estimation" for the name of the topic because that seem to be the prevalent term.  Even things by like `estimatesmartfee` that return feerates don't include "rate" in their name.

If you want, I can change the name of the article.  Alternatively, I can define the term here as a feerate times an estimated transaction size, but I think the main thing we want to cover is estimating feerates.  The process of multiplying them by estimated transaction size is less interesting to me.

Reply:
That’s alright, it’s just a pet-peeve of mine that across the Bitcoin Core codebase feerates are often misnamed as "…fee" which adds to the initial hurdle of trying to get familiar with what’s going on. "Fee estimation" is indeed the prevalent term